src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/OrderStateController.php

```diff
@@ -72,6 +72,11 @@ public function indexAction(
             'help_link' => $this->generateSidebarLink($request->attributes->get('_legacy_controller')),
             'orderStatesGrid' => $this->presentGrid($orderStatesGrid),
             'orderReturnStatesGrid' => $this->presentGrid($orderReturnStatesGrid),
+            'multistoreForcedContextInfoTip' => $this->trans(
```

Comment:
I think the common message is enough `getMultistoreAllShopsContextForcedMessage`
But I didn't read the specs :D

Reply:
Hello @sowbiba and @Progi1984 the 2 messages are different, as mentioned in the specs 😅 

Message for listing:

> [...] , this is why your context has just switched.

Message for BO page that allows to modify data

> [...]. It will be added to all your stores.